UI/window-basic-main.cpp

```diff
@@ -3621,7 +3635,7 @@ bool OBSBasic::QueryRemoveSource(obs_source_t *source)
 
 #define UPDATE_CHECK_INTERVAL (60 * 60 * 24 * 4) /* 4 days */
 
-#ifdef UPDATE_SPARKLE
+#if defined(ENABLE_SPARKLE_UPDATER)
```

Comment:
I'm not related to the project at all, but this appears to be a non-semantic refactor, and non-semantic refactors should be avoided in functional/semantic PRs because they tend to make the diff larger and the changes therefore much harder to assess.

Reply:
As [previously pointed out](https://github.com/obsproject/obs-studio/pull/5875#issuecomment-1023529237), this PR is based on PR #5155, which is the source of changes like this.